pkg/cmd/taskrun/describe.go

```diff
@@ -29,75 +29,81 @@ import (
 	cliopts "k8s.io/cli-runtime/pkg/genericclioptions"
 )
 
-const templ = `Name:	{{ .TaskRun.Name }}
-Namespace:	{{ .TaskRun.Namespace }}
+const templ = `{{decorate "bold" "Name"}}:	{{ .TaskRun.Name }}
+{{decorate "bold" "Namespace"}}:	{{ .TaskRun.Namespace }}
 {{- $tRefName := taskRefExists .TaskRun.Spec }}{{- if ne $tRefName "" }}
-Task Ref:    {{ $tRefName }}
+{{decorate "bold" "Task Ref"}}:    {{ $tRefName }}
 {{- end }}
 {{- if ne .TaskRun.Spec.ServiceAccountName "" }}
-Service Account:	{{ .TaskRun.Spec.ServiceAccountName }}
+{{decorate "bold" "Service Account"}}:	{{ .TaskRun.Spec.ServiceAccountName }}
 {{- end }}
 
-Status
+{{decorate "underline bold" "Status"}}
+
 STARTED 	DURATION 	STATUS
```

Comment:
Is an extra space needed here before STARTED/with data underneath?

Reply:
Looking at your screenshots helped clarify why. I see now it looks better this way.